curvine-client/src/unified/mod.rs

```diff
@@ -24,6 +24,9 @@ use std::collections::HashMap;
 #[cfg(feature = "opendal")]
 use curvine_ufs::opendal::*;
 
+#[cfg(feature = "oss")]
```

Comment:
The feature name conflicts with OSS; it is recommended to change it to oss-hdfs.

Reply:
Yes, it would be better to use oss-hdfs here. I plan to use the name "oss" next month when I integrate the official OSS Rust SDK.